### ISSUE TITLE
Crash on GuildBank Money Deposit

### ISSUE THREAD

**Issue by wowpide** [azerothcore/azerothcore-wotlk#2504]
Crash on GuildBank Deposit of Gold

##### SMALL DESCRIPTION:
deposit gold into guildbank

##### EXPECTED BLIZZLIKE BEHAVIOUR:
money goes in and is logged

##### CURRENT BEHAVIOUR:
server crash's 
```
2019-12-14 04:23:33 SQL: An error occurred formatting string "INSERT INTO log_money VALUES(%u, %u, "%s", "%s", %u, "%s", %u, "<GB WITHDRAW> %s (guild id: %u, members: %u, new amount: %I64u, leader guid low: %u, char level: %u)", NOW())" : invalid type specifier
2019-12-14 04:23:33 ERROR: [1064] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'An error occurred formatting string "INSERT INTO log_money VALUES(%u, %u, "%s", ' at line 1
```

##### STEPS TO REPRODUCE THE PROBLEM:
1. Create Guild
2. Deposit Money

##### EXTRA NOTES:
Didnt happen on MariaDB only on MySQL
##### BRANCH(ES):

master

##### AC HASH/COMMIT:
https://github.com/azerothcore/azerothcore-wotlk/commit/5e2ac178887ffe8cf322cdd4036acfeab2db4990

##### OPERATING SYSTEM:
Windows 10

##### MODULES:
no

##### OTHER CUSTOMIZATIONS:
no

**Comment by Shard-MW** [comment]
Confirmed

**Comment by Helias** [comment]
@wowpide test the fix and give us feedback ;-) https://github.com/azerothcore/azerothcore-wotlk/pull/2506

**Comment by Bogir** [comment]
no errors on Win7 MySQL

**Comment by wowpide** [comment]
fix #2506 works top Job
